libraries/botbuilder-applicationinsights/src/applicationInsightsTelemetryClient.ts

```diff
@@ -94,33 +97,56 @@ export class ApplicationInsightsTelemetryClient implements BotTelemetryClient, B
         return this.config;
     }
 
-    /* defaultClient()
+    /**
      * Provides direct access to the telemetry client object, which might be necessary for some operations.
      */
     get defaultClient(): appInsights.TelemetryClient {
         return this.client;
     }
 
+    /**
+     * Sends information about an external dependency (outgoing call) in the application.
+     * @param telemetry Telemetry Dependency.
```

Comment:
I think we want to use `@link` when referring to types. @stevengum @JonathanFingold can one of you weigh in on this?

Reply:
I'm not sure I understand, this is what I've seen in the past and is throughout the codebase. Are you asking about using `[link-text](xref:uid)` to cross-link concepts found in the code? 

```ts
// Example snippet of using xref
/**
 * A [BotAdapter](xref:botbuilder-core.BotAdapter) that can connect a bot to a service endpoint.
 * Implements [IUserTokenProvider](xref:botbuilder-core.IUserTokenProvider).
 */
```

To my knowledge we haven't used xrefs in the `@param` fields, is that supported @JonathanFingold?

Reply:
`@link` links don't work with our publishing build tools. use `[link text](xref:uid)` style links.

To find the uid for JavaScript, go to https://docs.microsoft.com/en-us/javascript/api/ and search for the member you want to link to. The uid will be the link text, as opposed to the link target. For example, the xref uid for the **BotAdapter.use** method is **botbuilder-core.BotAdapter.use**.

Reply:
For @param entries, the build tools have a bug and the parameter type is rendered but not linked.

Please describe the parameter in a way that is not simply stating what its parameter type is. For the **trackDependency** method, I have no idea what the telemetry dependency argument is for, so that would be good to explain here.

It would probably be useful to our readers to have links in the parameter descriptions to the parameter type (if the type is a Bot Framework SDK type). In this case, the link would be `[TelemetryDependency](xref:botbuilder-core.TelemetryDependency)`.